process/block/baseProcess_test.go

```diff
@@ -655,14 +655,6 @@ func TestCheckProcessorNilParameters(t *testing.T) {
 			},
 			expectedErr: process.ErrNilHeaderIntegrityVerifier,
 		},
-		{
-			args: func() blproc.ArgBaseProcessor {
-				args := createArgBaseProcessor(coreComponents, dataComponents, bootstrapComponents, statusComponents)
-				args.EpochNotifier = nil
-				return args
-			},
-			expectedErr: process.ErrNilEpochNotifier,
-		},
 		{
```

Comment:
can we add nil checks for EpochNotifier and EnableEpochHandler which are in coreComponents now?